uPortal-webapp/src/main/webapp/scripts/session-timeout.js

```diff
@@ -122,8 +122,8 @@ up.SessionTimeout =
              * Update the countdown time on the dialog.  If countdown
              * reaches 0, will auto-logout.
              */
-            updateCountdown = function() {
-                var now = new Date().getTime();
+            updateCountdown = function () {
+                var now = Date.now();
```

Comment:
automatically updated, same timestamp, but avoids instantiating a new class to generate it.